Updating maven-failsafe-plugin to 3.0.0-M5 fails CI integration tests under openjdk11

Not sure about this one ... something wrong with the classpath definition wrt JDK11 modules?
```
[INFO] --- maven-failsafe-plugin:3.0.0-M5:integration-test (default) @ disq ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.disq_bio.disq.RealWorldFilesIT
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/travis/.m2/repository/org/apache/spark/spark-unsafe_2.12/3.0.1/spark-unsafe_2.12-3.0.1.jar) to constructor java.nio.DirectByteBuffer(long,int)
[ERROR] WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
[ERROR] Tests run: 139, Failures: 0, Errors: 139, Skipped: 0, Time elapsed: 7.313 s <<< FAILURE! - in org.disq_bio.disq.RealWorldFilesIT
[ERROR] org.disq_bio.disq.RealWorldFilesIT.testReads(/home/travis/build/disq-bio/disq/gatk/src/test/resources/large/expected.NA12878.RNAseq.splitNcigarReads.noSecondaryAlignments.bam, /home/travis/build/disq-bio/disq/gatk/src/test/resources/large/human_g1k_v37.20.21.fasta, 1048576, false) [0]  Time elapsed: 0.016 s  <<< ERROR!

java.lang.NoClassDefFoundError: htsjdk/samtools/ValidationStringency
	at disq@0.3.7-SNAPSHOT/org.disq_bio.disq.RealWorldFilesIT.testReads(RealWorldFilesIT.java:70)

Caused by: java.lang.ClassNotFoundException: htsjdk.samtools.ValidationStringency
	at disq@0.3.7-SNAPSHOT/org.disq_bio.disq.RealWorldFilesIT.testReads(RealWorldFilesIT.java:70)
```
https://travis-ci.org/github/disq-bio/disq/jobs/756879092

--- a/pom.xml
+++ b/pom.xml
@@ -306,7 +306,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
-                <version>3.0.0-M4</version>
+                <version>3.0.0-M5</version>
                 <executions>
                     <execution>
                         <goals>

